src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <div>
+        <img
+          src="images/Search.svg"
+          alt="bigSerch"
```

Comment:
Use understandable alt attributes 
```suggestion
          alt="big search"
```